.gitignore

```diff
@@ -0,0 +1,3 @@
+/.idea
+/target
+/Cargo.lock
```

Comment:
You want the Cargo.lock for bins? Probably another reason to go for the workspace.